Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'tic_tac_toe_core', '0.1.0', :source => 'https://desolate-caverns-2106.herokuapp.com/'
+gem 'tic_tac_toe_core', '0.2.3', :source => 'https://desolate-caverns-2106.herokuapp.com/'
```

Comment:
That's a weird version jump: 0.1.0 => 0.2.3. Why not 0.1.1 or 0.2.0, or 1.0.0? Search for semantic versioning for more to read about this.